Source/WebKit/DerivedSources.make

```diff
@@ -510,6 +513,38 @@ all : $(AUTOMATION_PROTOCOL_OUTPUT_FILES)
 
 all : WebAutomationSessionProxyScriptSource.h
 
+# Much like Web Inspector protocol, WebDriver Bidi protocol consists of multiple domains. Each domain has one .json specification file.
+BIDI_PROTOCOL_INPUT_FILES = \
```

Comment:
ditto with all of these (especially to prevent confusion with bidi text)

perhaps these could be `AUTOMATION_BIDI_*` or `WEBDRIVER_BIDI_*`?

Reply:
WEBDRIVER_BIDI it is..